src/store.js

```diff
@@ -34,6 +34,8 @@ const state = {
     xb1: [],
     switch: [],
   },
+  poeMapToggles: {},
```

Comment:
something small, could we populate them by default with the labels (at least) for the maps so they don't show up blank?

Reply:
@TobiTenno done!